hugo-extended.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: hugo-extended
-  version: 0.125.5
+  version: 0.125.6
```

Comment:
We are missing the commit here.

Reply:
This package is using go/install so Commit is not needed here